src/System.Private.ServiceModel/tests/Common/Infrastructure/src/testproperties.props

```diff
@@ -86,26 +86,39 @@
       <NegotiateTestUpn>testuser@DOMAIN.CONTOSO.COM</NegotiateTestUpn>
     </PropertyGroup>
 
+    <!-- Convention for TestProperties values to use by [ConditionalFact]:
+      'true' = yes, the condition is satisfied
+      'false' = no, the condition is not satisfied
+      empty string = self-detect at runtime if possible, else assume 'false'
+    -->
     <PropertyGroup Condition="'$(Domain_Joined)' == ''">
-      <Domain_Joined>false</Domain_Joined>
+      <Domain_Joined></Domain_Joined>
```

Comment:
Reviewers -- a blank value now means "auto detect please".  Some auto-detection is over-simplified to get this to work across our platforms, but it will mature into real tests at some point.